tests/installation/playwright.config.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import path from 'path';
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { config as loadEnv } from 'dotenv';
+loadEnv({ path: path.join(__dirname, '..', '..', '.env') });
+
+const config: PlaywrightTestConfig = {
+  testIgnore: '**\/fixture-scripts/**',
+  globalSetup: path.join(__dirname, 'globalSetup'),
+  timeout: 5 * 60 * 1000,
+  retries: 0,
+  reporter: process.env.CI ? 'dot' : [['list'], ['html', { open: 'on-failure' }]],
+  forbidOnly: !!process.env.CI,
+  workers: 1,
```

Comment:
Parallel doesn't quite work…so we pin to `workers: 1`…for now! Post-merge, we can dig in!